<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="12">
     <widget class="QLabel" name="label_8">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>6)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Палиндром</string>
      </property>
     </widget>
    </item>
    <item row="2" column="13" colspan="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Округление числа</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="1" column="6" colspan="3">
     <widget class="QLineEdit" name="lineEdit">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="pushButton_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Количество гласных</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9" colspan="5">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_5">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>3)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>1)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="13" colspan="2">
     <widget class="QTextBrowser" name="textBrowser_6">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ввести строку кириллицей (2,3,4,5)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="11">
     <widget class="QTextBrowser" name="textBrowser_5">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="14">
     <widget class="QLabel" name="label_10">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ввести K (6)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="14">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>2)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="11">
     <widget class="QPushButton" name="pushButton_5">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Количество букв</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8" colspan="2">
     <widget class="QTextBrowser" name="textBrowser_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QLabel" name="label_7">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>5)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8" colspan="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Количество согласных</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QLabel" name="label_9">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ввести N (6)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_6">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>4)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Алгоритм Цезаря</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Введите строку латиницей (1)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_2">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ввести U (1)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
